C-Plus-Plus/recursion/Letter_Case_Permutation_using_recursion.cpp

```diff
@@ -0,0 +1,64 @@
+/*Letter Case Permutation using recursion
+
+Given a string S, we can transform every letter individually to be lowercase or uppercase to create another string.
+Print all possible strings we could create. return the output in any order. */
+
+
+#include <iostream>
+using namespace std;
+
+void solve(string output, string str)
+{
+    if (str.length() == 0)
+    {
+        cout << output << " ";
+        return;
+    }
+
+    if (isalpha(str[0]))
+    {
+
+        string output1 = output;
+        op1.push_back(tolower(str[0]));
+        string output2 = output;
+        op2.push_back(toupper(str[0]));
+
+        str.erase(str.begin() + 0);
+
+        solve(output1, str);
+        solve(output2, str);
+    }
+    else
+    {
+        string output3 = output;
+        op3.push_back(str[0]);
+
+        str.erase(str.begin() + 0);
+
+        solve(output3, str);
+    }
+}
+
+int main()
+{
+    string str;
+    cout << "Enter an alphanumeric string : "
```

Comment:
@1711shashank Expected `;` here.

Reply:
Sorry, 
By Mistake i have added the different/untested files